src/ParkChecklist/ParkChecklist.js

```diff
@@ -123,161 +93,120 @@ const ParkChecklist = ({ apiLink }) => {
-            )}
-          </div>
-        </>
-      ),
-    },
-    {
-      path: `/login`,
-      element: (
-        <div>
-          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn} />
+  return (
+    <div className="park-checklist-container">
+      {loading ? <p className="loading-message">Loading Parks...</p> : (
+        <section>
+      <div className="below-header-box">
+        <div className="state-selector">
+          <select onChange={handleStateChange} value={selectedState}>
+            <option value="">Select a State</option>
+            {states.map((stateCode) => (
+              <option key={stateCode} value={stateCode}>
+                {stateCode}
+              </option>
+            ))}
+          </select>
+          <button onClick={handleGoClick}>Go!</button>
         </div>
-      )
-    },
-    {
-      path: `/Parks/${selectedState}`,
-      element: showResults ? (
-        <SelectedStateParks
-          parks={parks}
-          selectedState={selectedState}
-          fetchParkDetails={fetchParkDetails}
+      </div>
+      <div className="sorting-options">
+        <div className="sort-by-state">
+          <StateSort
+            states={states}
+            selectedStateParks={selectedStateParks}
+            handleStateCheckboxChange={handleStateCheckboxChange}
+          />
+        </div>
+        <DesignationSort
+          getUniqueDesignations={getUniqueDesignations}
+          selectedDesignations={selectedDesignations}
+          handleDesignationCheckboxChange={handleDesignationCheckboxChange}
         />
-      ) : null,
-    },
-    {
-      path: `/Parks/${selectedState}/${park_code}`,
-      element: selectedState ? <ParkDetails foundPark={foundPark} /> : null,
-    },
-    {
-      path: "/*",
-      element: <Navigate to="/" />,
-    },
-  ]);
-
-  return (
-    <div
-      className="park-checklist-container"
-      // style={{ backgroundImage: `url(${page})` }}
-    >
-      <Link to="/" className="header-link">
-        <h1 className="page-title">National Park Service Service</h1>
-      </Link>
-      {loading ? <p className="loading-message">Loading Parks...</p> : routes}
+      </div>
+      <h2>{`${totalChecked}/${parks.length} parks visited`}</h2>
+      <h2>Parks Checklist</h2>
+      <ul className="checkbox-list">
+        {selectedFilters.parks.length > 0 ? (
+          selectedFilters.parks
+            .filter(
+              (park) =>
+                selectedFilters.designations.length === 0 ||
+                selectedFilters.designations.some(
+                  (designation) =>
+                    park.attributes.designation === designation
+                )
+            )
+            .map((park) => (
+              <li key={park.id}>
+                <input
+                  type="checkbox"
+                  id={park.id}
+                  checked={checkedItems.includes(park.id)}
+                  onChange={() => handleCheckboxChange(park.id)}
```

Comment:
would it be worthwhile to turn the pagination into its own component? I can do that tonight